src/Emogrifier/CssInliner.php

```diff
@@ -220,15 +215,17 @@ public function getDomDocument()
     }
 
     /**
-     * Sets the CSS to merge with the HTML.
+     * Inlines the given CSS into the existing HTML.
      *
-     * @param string $css the CSS to merge, must be UTF-8-encoded
+     * @param string $css the CSS to inline, must be UTF-8-encoded
      *
-     * @return void
+     * @return CssInliner fluent interface
      */
-    public function setCss($css)
+    public function inlineCss($css = '')
     {
-        $this->css = $css;
+        $this->process($css);
+
+        return $this;
     }
```

Comment:
Could we dispense with this method and instead rename `process()` to `inlineCss()` (and have it `return $this` for chaining)?

Reply:
Done.